### PR TITLE
UIDEXP-372: Apply tenant date-time when filter data export logs by date on View all page

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
https://folio-org.atlassian.net/browse/UIDEXP-372
Here we made changes of building query based on tenant timezone.

### CODE REVIEW

File: src/components/AllJobLogsView/AllJobLogsView.js [hunk 1]
```diff
@@ -51,48 +51,50 @@ const initialQuery = '?qindex=hrID&sort=-completedDate';
...
-          return jobStatusFailString;
-        case Array.isArray(query):
-          return `status=(${query.map(v => (
-            v === JOB_EXECUTION_STATUSES.COMPLETED
-              ? JOB_LOGS_STATUS_QUERY_VALUE
-              : `"${v}"`)).join(' or ')})`;
-        default:
-          return `status=${query}`;
-      }
+const buildJobsQuery = (timezone) => {
```
Comment (NikitaSedyx) [inline src/components/AllJobLogsView/AllJobLogsView.js:54]: is it possible to avoid this change?

Comment (UladzislauKutarkin) [reply src/components/AllJobLogsView/AllJobLogsView.js:54]: let me check, if it possible to get access to timezone in other way

File: src/components/AllJobLogsView/AllJobLogsView.js [hunk 2]
```diff
@@ -51,48 +51,50 @@ const initialQuery = '?qindex=hrID&sort=-completedDate';
...
-    '-updated': '-updatedDate',
-    jobProfileName: 'jobProfileName',
-    '-jobProfileName': '-jobProfileName',
-    runBy: 'runByFirstName runByLastName',
-    '-runBy': '-runByFirstName runByLastName',
-    startedDate: 'startedDate',
-    '-startedDate': '-startedDate',
-  }
-);
+    {
```
Comment (vashjs) [inline src/components/AllJobLogsView/AllJobLogsView.js:78]: Please revert this formatting, let's review our lint configuration first and allign with all other modules

File: src/components/AllJobLogsView/AllJobLogsView.js (hunk 1 above)
Comment (UladzislauKutarkin) [reply src/components/AllJobLogsView/AllJobLogsView.js:54]: I haven't found another way to get access to stripes.timzone

File: src/components/AllJobLogsView/AllJobLogsView.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply src/components/AllJobLogsView/AllJobLogsView.js:78]: @vashjs , all rules are taken from `@folio/eslint-config-stripes`. We don't have any local rules for lint in` ui-data-export`

**Review by vashjs — APPROVED** [review]

**Review by OleksandrHladchenko1 — APPROVED** [review]

**Pull request merged**